src/app/candidate.ts

```diff
@@ -0,0 +1,15 @@
+export class Candidate {
+    // Resume JSON Schema:
+    // https://jsonresume.org/schema/
+    constructor(
+        public name: string,
+        public label: string,
+        public email: string,
+        public phone: string,
+        public website: string,
+        public github: string,
+        public twitter: string,
```

Comment:
I've noticed that you've pre-defined twitter here. I'm not familiar with angular but have you accounted for the case of adding multiple accounts? (ie. Facebook, Instagram, etc)